tests/python_package_test/test_sklearn.py

```diff
@@ -719,6 +719,24 @@ def test_predict():
     with pytest.raises(AssertionError):
         np.testing.assert_allclose(res_engine, res_sklearn_params)
 
+    # Test multiclass binary classification
+    num_samples, num_classes = 5, 2
+    X_train = np.ones((num_samples, 3))
+    y_train = np.zeros((num_samples,))
```

Comment:
```suggestion
    num_samples = 100
    num_classes = 2
    X_train = np.linspace(start=0, stop=10, num=num_samples*3).reshape(num_samples, 3)
    y_train = np.repeat([0, 1], repeats=num_samples/2)
```

It's a point of minor personal style preference, but could you please avoid using one line to assign multiple constants?

And could you please follow this suggestion to use 100 samples instead of 5, and a target that actually has samples from 2 classes, so some minimal training actually happens?

Reply:
Thanks, fix it as you suggested in bbbd5ba and 57e509c